doc/HOWTO-KAFKA.md

```diff
@@ -37,9 +37,14 @@ Test notes with standard Kafka server, see https://kafka.apache.org/quickstart.
 
 #### Setup
 
+Download a [Kafka Binary Release](https://dlcdn.apache.org/kafka/3.9.0/kafka_2.13-3.9.0.tgz) and unpack it:
```

Comment:
For transparency and some resilience, could we instead link to https://kafka.apache.org/downloads and state in text that one should for now download the 3.9.0 release as that's the only that is tested and supported?